app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] == "mv":
+        cmd, source_file, destination_path = parts
+
+    if os.path.exists(source_file):
+        if os.path.isdir(destination_path):
+            os.makedirs(destination_path, exist_ok=True)
+            destination_file = os.path.join(destination_path,
+                                            os.path.basename(source_file))
+
+        else:
+            if os.path.isdir(destination_path):
+                destination_file = os.path.join(destination_path,
+                                                os.path.basename(source_file))
+            else:
+                destination_file = destination_path
+
+                os.replace(source_file, destination_file)
```

Comment:
Command `move_file("mv file.txt first_dir/second_dir/third_dir/file2.txt")` does not work as expected in case directories are absent. Resolve this issue